app/src/main/java/org/oppia/app/player/state/DragDropSortInteractionView.kt

```diff
@@ -62,26 +76,52 @@ class DragDropSortInteractionView @JvmOverloads constructor(
         },
         bindView = { view, viewModel ->
           val binding = DataBindingUtil.findBinding<DragDropInteractionItemsBinding>(view)!!
+          binding.dragDropItemRecyclerview.adapter = createNestedAdapter()
+          binding.adapter = adapter
+          binding.dragDropContentGroupItem.isVisible = isMultipleItemsInSamePositionAllowed
+          binding.dragDropContentUnlinkItems.isVisible = viewModel.htmlContent.htmlList.size > 1
+          binding.viewModel = viewModel
+        }
+      )
+      .build()
+  }
+
+  private fun createNestedAdapter(): BindableAdapter<String> {
+    return BindableAdapter.SingleTypeBuilder
+      .newBuilder<String>()
+      .registerViewBinder(
+        inflateView = { parent ->
+          DragDropSingleItemBinding.inflate(
+            LayoutInflater.from(parent.context), parent, /* attachToParent= */ false
+          ).root
+        },
+        bindView = { view, viewModel ->
+          val binding = DataBindingUtil.findBinding<DragDropSingleItemBinding>(view)!!
           binding.htmlContent =
-            htmlParserFactory.create(resourceBucketName, entityType, entityId, /* imageCenterAlign= */ false)
+            htmlParserFactory.create(
+                resourceBucketName,
+                entityType,
+                entityId, /* imageCenterAlign= */
```

Comment:
imageCenterAlign= should go in the next line (along with the false)

Reply:
done